be/src/exec/vectorized/aggregate/aggregate_streaming_node.cpp

```diff
@@ -72,11 +72,11 @@ Status AggregateStreamingNode::get_next(RuntimeState* state, ChunkPtr* chunk, bo
                 if (false) {
                 }
 #define HASH_MAP_METHOD(NAME)                                                                                          \
-    else if (_aggregator->hash_map_variant().type == HashMapVariant::Type::NAME) {                                     \
+    else if (_aggregator->hash_map_variant().type == AggHashMapVariant::Type::NAME) {                                  \
         TRY_CATCH_BAD_ALLOC(_aggregator->build_hash_map<decltype(_aggregator->hash_map_variant().NAME)::element_type>( \
                 *_aggregator->hash_map_variant().NAME, input_chunk_size));                                             \
     }
-                APPLY_FOR_VARIANT_ALL(HASH_MAP_METHOD)
+                APPLY_FOR_AGG_VARIANT_ALL(HASH_MAP_METHOD)
```

Comment:
Could this rename work be moved to another PR?

Reply:
Ok, I'll do it